src/OrbitGl/Button.cpp

```diff
@@ -28,6 +28,17 @@ void Button::SetLabel(const std::string& label) {
   RequestUpdate(RequestUpdateScope::kDraw);
 }
 
+void Button::SetMouseReleaseCallback(MouseReleaseCallback callback) {
+  mouse_release_callback_ = callback;
+}
+
+void Button::OnRelease() {
+  CaptureViewElement::OnRelease();
+  if (mouse_release_callback_ != nullptr) {
+    mouse_release_callback_(this);
+  }
```

Comment:
I'm wondering - considering symmetry - if this call should be before calling `CaptureViewElement::OnRelease`.

Not sure if that makes sense though.

Reply:
I actually think it makes more sense this way around, because `CaptureViewElement` might do some cleanup to leave the "mouse pressed" state of the button, and I think the callback should be fired at the end of releasing the button. You could argue it should be called "AfterRelease" then...